Dockerfile

```diff
@@ -1,41 +1,39 @@
-FROM ubuntu:22.04
-USER root
 ARG DEBIAN_FRONTEND=noninteractive
-
 # update and upgrade packages
 RUN apt-get -y update --fix-missing \
     && apt-get -y upgrade \
-    && apt-get -y update \
-    && apt-get -y autoremove
-
+    && apt-get -y install software-properties-common \
+    && add-apt-repository ppa:deadsnakes/ppa \
+    && apt-get -y update
 # install dependencies
-RUN apt-get -y install \
+RUN apt-get -yqq install \
+    fontconfig \
     git \
-    python3.9 \
+    libglib2.0-0 \                                                                      
+    libsm6 \                                                                            
+    libxrender1 \                                                                       
+    libxext6 \
+    libfreetype6 \                                                                      
+    libgomp1 \
+    python3.12 \                                                                        
     python3-pip \
     tar \
-    unzip \
-    wget \
-    openjdk-11-jdk \
-	vim \
-	dotnet-runtime-6.0
-	
+    unzip \ 
+    wget
 # install python packages
 RUN pip uninstall easypqp \
     && pip install git+https://github.com/Nesvilab/easypqp.git@master \
-    && pip install lxml
-	
-# set the working directory to the HOME of root
-WORKDIR /root
-
-# create directories
-RUN mkdir tmp
-	
+    && pip install lxml \
+    && pip install --force-reinstall numpy==1.25 \
+    && pip install --force-reinstall pandas==1.5.3 \
+    && pip install pyopenms
+# Install OpenJDK-8
+RUN apt-get update && \
+    apt-get install -y openjdk-18-jdk && \
```

Comment:
The original dockfile has Java 11 installed. We never tested Java 18 thoroughly.